dex-auth/dex-crds/overlays/github/params.yaml

```diff
@@ -0,0 +1,5 @@
+varReference:
+- path: data/config.yaml
+  kind: ConfigMap
+- path: spec/http/route/destination/host
+  kind: VirtualService
```

Comment:
nit: add a new line for EOF?

Reply:
see https://github.com/kubeflow/manifests/pull/1202/commits/57166db1933d219273e4d0f8891a22b653ab3acb